src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -206,6 +206,10 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
 {
     const HWIntrinsic intrin(node);
 
+    // We need to validate that other phases of the compiler haven't introduced unsupported intrinsics
+    CORINFO_InstructionSet isa = HWIntrinsicInfo::lookupIsa(intrin.id);
+    assert(compiler->compIsaSupportedDebugOnly(isa));
```

Comment:
```suggestion
    assert(compiler->compIsaSupportedDebugOnly(HWIntrinsicInfo::lookupIsa(intrin.id)));
```
to avoid unusual code in release. (C++ compiler may optimize it out, but one never knows.)